lib/plugin/instance.rb

```diff
@@ -1032,6 +1032,50 @@ def register_email_unsubscriber(type, unsubscriber)
     DiscoursePluginRegistry.register_email_unsubscriber({ type => unsubscriber }, self)
   end
 
+  # Allows the plugin to export additional site stats via the About class
+  # which will be shown on the /about route. The stats returned by the block
+  # should be in the following format (these four keys are _required_):
+  #
+  # {
+  #   last_day: 1,
+  #   7_days: 10,
+  #   30_days: 100,
+  #   count: 1000
```

Comment:
Count is too generic imo, I'm guessing this refers to the life time count?

Reply:
It's the total count and all of these keys are used because that's what all the existing ones are like:

```
topic_count: 27480,
post_count: 490358,
user_count: 41719,
```

I don't want to use different terminology here.

Reply:
I guess this will be resolve soon since the terms here will have a prefix.

Reply:
Well the keys provided from the plugin will still be in this format, but yes the output from the endpoint will have the prefix now and be on the top level.